submissions/BlueLamantine/html-css-popup/index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Hooli-style Popup</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="shortcut icon" href="img/favicon.png" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <nav class="topnav">
+      <a href="#">Mail</a>
+      <a href="#">Images</a>
```

Comment:
Try to use more semantic tags.
```
<nav>
    <ul>
        <li>
            <a href="#"></a>
        </li>
        <li>
            <a href="#"></a>
        </li>
    </ul>
</nav>
```

Reply:
OK, why not! I rewrited html-stucture of page with semantic tags <ul><li>, all links nested into list items.